src/config/configJsonSchema.js

```diff
@@ -232,6 +232,10 @@ module.exports = {
                   required: ['port'],
                   additionalProperties: false,
                 },
+                genesisTime: {
```

Comment:
I would probably group all options from `genesis.json` into `genesis` object. Coz most of them actually changing in time, but we need to specify genesis one (how they were on block 1) to be able to sync.

Reply:
You mean every option specified in the file? Or just the validators and chain_id? I don't know anything about tendermint, so I don't understand what happens when checking the genesis or what other options might be variable.

Reply:
Yeah, we need all of them in the config. So we can define the `genesis` property as an object that should accept the whole object from `genesis.json` file.

Reply:
Should we validate the JSON with Ajv still, or treat it like a string? I can't get doT to interpolate an entire JSON value, it just returns `[object Object]`...

Reply:
I think we don't need to validate it but just kept it there as an JSON object (not serialized to string). Just use `type: object` and do not define `properties`. May be in this case it's better to take this info and put result of `JSON.stringify` directly to the `genesys.json` file. So basically we don't need the template for this case.

Reply:
OK, found my syntax error with doT. We now treat it as an object and the template evaluates the entire object through JSON.stringify(), so it can be handled in the same way as the other templated files.

Reply:
Great!